runtime-sdk/src/crypto/signature/secp256r1.rs

```diff
@@ -41,22 +41,10 @@ impl PublicKey {
         message: &[u8],
         signature: &Signature,
     ) -> Result<(), Error> {
-        // Note that we must use Sha512_256 instead of our Hash here,
```

Comment:
The comment didn't seem to match the code, so I updated it to match the p384 implementation.

Reply:
What do you mean it didn't reflect the code? The new implementation is better though, but it still must use `Sha512_256` instead of our `Hash` for the reason it was mentioned earlier.

Reply:
I guess I didn't understand what `Hash` meant since it wasn't defined in the file and I couldn't tell what it's the "same thing" as.